packages/x-components/src/x-modules/facets-next/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { isStringEmpty } from '../../utils/string';
```

Comment:
Should you move the tests of this function too?

Reply:
done